image/descriptor.go

```diff
@@ -48,19 +50,36 @@ func (d *descriptor) hash() string {
 	return pts[1]
 }
 
+// Index implementation
+type Index v1.ImageIndex
```

Comment:
This is not really needed. Is there a particular reason to make a local type for this struct? I could end up complicating golang type assertions